attributes/default.rb

```diff
@@ -3,17 +3,20 @@
 
 case node['platform']
 when 'debian', 'ubuntu'
-  default['pgpool']['config']['package_name'] = 'pgpool2'
+  default['pgpool']['config']['package_name'] = 'pgpool-II-pg93' # Might override
   default['pgpool']['config']['dir'] = '/etc/pgpool2'
   default['pgpool']['service'] = 'pgpool2'
 when 'centos'
-  default['pgpool']['config']['package_name'] = 'postgresql-pgpool-II'
+# Old default package name was installing an ancient Pgpool version. As such,
+# increment the default here such that it installs the version meant for
+# Postgres-9.3.x, regardless this is overrideable like any other attribute.
+default['pgpool']['config']['package_name'] = 'pgpool-II-pg93'
   default['pgpool']['config']['dir'] = '/etc/pgpool-II'
   default['pgpool']['service'] = 'pgpool'
 end
 
 default['pgpool']['pgconf']['listen_addresses'] = 'localhost'
-default['pgpool']['pgconf']['port'] = 5432
+default['pgpool']['pgconf']['port'] = 9999
```

Comment:
Given that these are Postgres clients that would be connecting, I saw no reason to leave that particular default at 9999 given that running a DB plus Pgpool on the same machine isn't exactly encouraged.